atlasdb-impl-shared/src/test/java/com/palantir/atlasdb/keyvalue/api/watch/ClientLogEventsTest.java

```diff
@@ -66,15 +75,26 @@ public final class ClientLogEventsTest {
 
     private static final LockWatchReference REFERENCE_1 = LockWatchReferences.entireTable("table.one");
 
+    private static final ChangeMetadata CHANGE_METADATA_A = ChangeMetadata.created(PtBytes.toBytes("aaa"));
+    private static final ChangeMetadata CHANGE_METADATA_B = ChangeMetadata.deleted(PtBytes.toBytes("bbb"));
+    private static final ChangeMetadata CHANGE_METADATA_C =
+            ChangeMetadata.updated(PtBytes.toBytes("cc"), PtBytes.toBytes("ccc"));
+    private static final LockRequestMetadata METADATA_1 =
+            LockRequestMetadata.of(ImmutableMap.of(DESCRIPTOR_1, CHANGE_METADATA_A));
+    private static final LockRequestMetadata METADATA_2 =
+            LockRequestMetadata.of(ImmutableMap.of(DESCRIPTOR_1, CHANGE_METADATA_B));
+
     private static final LockWatchEvent LOCK_WATCH_EVENT_VERSION_1 = LockWatchCreatedEvent.builder(
                     ImmutableSet.of(REFERENCE_1), ImmutableSet.of(DESCRIPTOR_3))
             .build(SEQUENCE_1);
-    private static final LockWatchEvent LOCK_DESCRIPTOR_2_VERSION_2 =
-            LockEvent.builder(ImmutableSet.of(DESCRIPTOR_2), LOCK_TOKEN_1).build(SEQUENCE_2);
+    private static final LockWatchEvent LOCK_DESCRIPTOR_2_VERSION_2 = LockEvent.builder(
+                    ImmutableSet.of(DESCRIPTOR_2), LOCK_TOKEN_1, Optional.of(METADATA_1))
+            .build(SEQUENCE_2);
     private static final LockWatchEvent UNLOCK_DESCRIPTOR_2_VERSION_3 =
             UnlockEvent.builder(ImmutableSet.of(DESCRIPTOR_1)).build(SEQUENCE_3);
-    private static final LockWatchEvent LOCK_DESCRIPTOR_1_VERSION_4 =
-            LockEvent.builder(ImmutableSet.of(DESCRIPTOR_1), LOCK_TOKEN_2).build(SEQUENCE_4);
+    private static final LockWatchEvent LOCK_DESCRIPTOR_1_VERSION_4 = LockEvent.builder(
+                    ImmutableSet.of(DESCRIPTOR_1), LOCK_TOKEN_2, Optional.of(METADATA_2))
+            .build(SEQUENCE_4);
```

Comment:
Adding metadata to these events just ensures that adding metadata does not break any of the existing tests.